scalding-core/src/main/scala/com/twitter/scalding/Execution.scala

```diff
@@ -48,6 +48,13 @@ sealed trait Execution[+T] extends java.io.Serializable {
   import Execution.{ EvalCache, FlatMapped, GetCounters, ResetCounters, Mapped, OnComplete, RecoverWith, Zipped }
 
   /**
+   * Lift an Execution into a Try
```

Comment:
can you explain that this Execution should never be failed? Only the Try might be failed.